src/Tribe/Front_Page_View.php

```diff
@@ -69,6 +72,70 @@ public function parse_query( WP_Query $query ) {
 		$query->is_singular = false;
 	}
 
+	/**
+	 * Parse the query when the customizer sends request to preview specifc page to avoid 404 pages
+	 * or the wrong page.
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_Query $query
+	 */
+	public function parse_customizer_query( $query ) {
+
+		$data = tribe_get_request_var( 'customized', array() );
+
+		if ( empty( $data ) ) {
+			return;
+		}
+
+		$data = json_decode( wp_unslash( $_REQUEST['customized'] ), true );
```

Comment:
I'm not familiar with `tribe_get_request_var` but is there some reason we can't do `json_decode( wp_unslash( $data ) )` here (or above)? 

Why the second step?

Reply:
You're right! we don't need it anymore as the value is on `$data`. Thanks  Sending the fix